userspace/libsinsp/filter.cpp

```diff
@@ -39,24 +39,9 @@ limitations under the License.
 ///////////////////////////////////////////////////////////////////////////////
 // sinsp_filter_expression implementation
 ///////////////////////////////////////////////////////////////////////////////
-sinsp_filter_expression::sinsp_filter_expression()
+void sinsp_filter_expression::add_check(std::unique_ptr<sinsp_filter_check> chk)
```

Comment:
Is this method really useful? It's a one liner...

Reply:
I see your point. I would vote to keep it for now as I prefer to move the expression checks as a private class member in the future rather than getting rid of its methods, regardless of how simple they are